### DISCUSSION TITLE
custom domain only going to snapshot homepage

### DISCUSSION THREAD

**Discussion by LunarWhaleDev** [snapshot-labs/snapshot#1105] [Q&A] [answered]
vote.thunderdao.com

I followed all the steps and had the custom domain merged into domains.json with the pull request
added to dns a cname record to point to snapshotpage.b-cdn.net

Is there another step I am missing to get it to load to my own page at https://snapshotpage.b-cdn.net/#/thunderdao.eth ?

perhaps I have made an error with the dns record?

Thanks in advance for any help

**Comment by metrox-eth** [comment]
It should be okay now, please retest !

**Reply by LunarWhaleDev** [reply]
wow!

thank you so much!
I was just so worried i didnt follow the steps properly

thank you !

**Comment by metrox-eth** [comment, marked as answer]
Custom domain PRs are merged at least once a day, however we still need to merge another module after that, it should not take more than 24/48h in total.

**Comment by BB-trapeza** [comment]
I met the same issue, and I have waited more than 2 days.
I set custom domain at snapshot setting (vote.trapeza.finance)
space PR merged well, added to DNS a cname record to point to snapshotpage.b-cdn.net, and request set domain. (https://ina9pk8175.execute-api.us-west-2.amazonaws.com/dev?domain=vote.trapeza.finance)

but custom domain to snapshot homepage. How can I fix it? Please help
